config/locales/server.en.yml

```diff
@@ -29,6 +29,24 @@ en:
       cannot_add_options: Cannot add options to this poll
       cannot_remove_options: Cannot remove options from this poll
       must_have_options: Must have at least one poll option
+    export:
+      percent_voted: “%{num}/%{denom} (%{percent}) members have stated their position”
+      title: "title"
+      author: "author"
+      created_at: "created_at"
```

Comment:
I think we want the translation strings in here to be as human-readable as possible (ie, 'Created at', rather than 'created_at'), since translators will be looking at this and may be confused by the underscore formatting (What's "outcome_created_at" in Spanish?).
```
# en.yml
poll:
  export:
    created_at: "Created at"
```
so that someone can translate to
```
# es.yml
poll:
  export:
    created_at: "Empezado"
```

My ideal thing here would be
- Create a hash where the keys are underscored symbols
```
{ poll_url: "loomio.org/p/key" }
```
- Translate that in the html template and as we're passing it into the csv
```
- keys.each do |key|
  %h2= t(:"poll.export.#{key}")
```
```
hash.each do |key, value|
  csv << [I18n.t(:"poll.export.#{key}"), value]
end
```

Reply:
I'm not too sure about this, I think I would like to provide the translation in the common table. Then I know what goes out will have translation and use the same key. 

I guess what you're also saying is that translation should be only done by replacing strings with I18n.t calls, not having intervening helper functions.